rllib/env/multi_agent_env_runner.py

```diff
@@ -157,9 +157,9 @@ def sample(
         """Runs and returns a sample (n timesteps or m episodes) on the env(s).
 
         Args:
-            num_timesteps: The number of timesteps to sample during this call.
+            num_timesteps: The minimum number of timesteps to sample during this call.
                 Note that only one of `num_timetseps` or `num_episodes` may be provided.
-            num_episodes: The number of episodes to sample during this call.
+            num_episodes: The minimum number of episodes to sample during this call.
```

Comment:
This is required because we don't want to leave episodes in a dangling state.
This dangling state would be reached if two episodes would be done at the same time, but one one of them would be replaced by a new episode. On the next call to env_runner.sample() with `force_reset=False`, we'd then try to add more steps to that dangling episode, causing an error because the agent would be done.